wcfsetup/install/lang/de.xml

```diff
@@ -3084,6 +3084,9 @@ Wenn {if LANGUAGE_USE_INFORMAL_VARIANT}du{else}Sie{/if} unter „Konfiguration 
 		<item name="wcf.acp.user.option.editable.2"><![CDATA[Administrator]]></item>
 		<item name="wcf.acp.user.option.editable.3"><![CDATA[Eigentümer und Administrator]]></item>
 		<item name="wcf.acp.user.option.editable.6"><![CDATA[Eigentümer bei Erstanmeldung und Administrator]]></item>
+		<item name="wcf.acp.user.option.labeledUrl"><![CDATA[URL]]></item>
+		<item name="wcf.acp.user.option.labeledUrl.description"><![CDATA[Wenn der Feldtyp „labeledUrl“ ist, muss hier die URL hinterlegt werden, auf die verlinkt werden soll. {if LANGUAGE_USE_INFORMAL_VARIANT}Nutze{else}Nutzen Sie{/if} <span class="inlineCode">%s</span> als Platzhalter für die Eingabe des Benutzers.]]></item>
+		<item name="wcf.acp.user.option.labeledUrl.error.invalid"><![CDATA[Die URL enthält nicht den Platzhalter <span class="inlineCode">%s</span>.]]></item>
```

Comment:
```suggestion
		<item name="wcf.acp.user.option.labeledUrl.error.invalid"><![CDATA[Die URL enthält den Platzhalter <span class="inlineCode">%s</span> nicht.]]></item>
```

Reply:
The suffixed "nicht" sounds very strange to me. I would therefore not implement your proposal.

Reply:
`Den Platzhalter <span class="inlineCode">%s</span> fehlt in der URL.`?